restforce.gemspec

```diff
@@ -20,7 +20,15 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '>= 0.8.8'
-  gem.add_dependency 'json', ['>= 1.7.5', '< 1.9.0']
+
+  # Ruby 2.4 requires JSON 2.0, but that doesn't work with old pre-2.0.0 versions of Ruby.
+  # See https://github.com/ejholmes/restforce/issues/260.
+  if RUBY_VERSION =~ /^1.9/
+    gem.add_dependency 'json', ['>= 1.7.5', '< 1.9.0']
+  else
+    gem.add_dependency 'json', '>= 2.0.0'
```

Comment:
this is a bit draconian. now if you're running say ruby 2.3, you _have_ to update json to 2.0.0 - which can be difficult (read: impossible) if you have other gems with the same problem of not allowing json 2.0